src/solver/variable/include/antares/solver/variable/info.h

```diff
@@ -26,8 +26,136 @@
+    {
+        const auto& thermal = results.data.area->thermal;
+        results.variableCaption = thermal.list.enabledClusterAt(i)->name();
+    }
+};
+
+// Specializations for Reserve Participation VCards
+template<>
+struct vcard_caption_traits<Economy::Reserves::VCardReserveParticipationByDispatchableOnUnitsPlant>
+{
+    static constexpr void apply(SurveyResults& results, uint i)
+    {
+        const auto& thermal = results.data.area->thermal;
+        auto [clusterName, reserveName] = thermal.list
+                                            .reserveParticipationClusterAt(results.data.area, i);
+        results.variableCaption = reserveName + "_" + clusterName;
+    }
+};
+
+template<>
+struct vcard_caption_traits<Economy::Reserves::VCardReserveParticipationByDispatchableOffUnitsPlant>
+{
+    static constexpr void apply(SurveyResults& results, uint i)
+    {
+        const auto& thermal = results.data.area->thermal;
+        auto [clusterName, reserveName] = thermal.list
+                                            .reserveParticipationClusterAt(results.data.area, i);
+        results.variableCaption = reserveName + "_" + clusterName + "_off";
+    }
+};
+
+template<>
+struct vcard_caption_traits<Economy::Reserves::VCardReserveParticipationByThermalGroup>
+{
+    static constexpr void apply(SurveyResults& results, uint i)
+    {
+        const auto& thermal = results.data.area->thermal;
+        auto [groupName, reserveName] = thermal.list.reserveParticipationGroupAt(results.data.area,
+                                                                                 i);
+        results.variableCaption = reserveName + "_" + groupName;
+    }
+};
+
+template<>
+struct vcard_caption_traits<Economy::Reserves::VCardReserveParticipationBySTStorage>
+{
+    static constexpr void apply(SurveyResults& results, uint i)
+    {
+        const auto& shortTermStorage = results.data.area->shortTermStorage;
+        auto [clusterName, reserveName] = shortTermStorage
+                                            .reserveParticipationClusterAt(results.data.area, i);
+        results.variableCaption = reserveName + "_" + clusterName;
+    }
+};
+
+template<>
+struct vcard_caption_traits<Economy::Reserves::VCardReserveParticipationBySTStorageGroup>
+{
+    static constexpr void apply(SurveyResults& results, uint i)
+    {
+        const auto& shortTermStorage = results.data.area->shortTermStorage;
+        auto [groupName, reserveName] = shortTermStorage
+                                          .reserveParticipationGroupAt(results.data.area, i);
+        results.variableCaption = reserveName + "_" + groupName;
+    }
+};
+
+template<>
+struct vcard_caption_traits<Economy::Reserves::VCardReserveParticipationByHydro>
+{
+    static constexpr void apply(SurveyResults& results, uint i)
+    {
+        const auto& hydro = results.data.area->hydro;
+        if (const auto reserveName = hydro.reserveParticipationAt(results.data.area, i))
```

Comment:
question: No caption at all in this case ? No fallback to default case "results.variableCaption = thermal.list.enabledClusterAt(i)->name();" ?